site/CookiePreferencesManager.tsx

```diff
@@ -168,8 +168,9 @@ export const parseDate = (date?: string): number | undefined => {
 
 export const getPreferenceValue = (
     type: PreferenceType,
-    preferences: Preference[]
+    preferences?: Preference[]
 ) => {
+    preferences ??= getInitialState().preferences
```

Comment:
Just checking, the only difference between this and a default parameter is that it coalesces `null` too?

Reply:
Yeah, there was no real reason for that really :)
It's just that at the time I didn't think about default parameters, but they're of course nicer!